examples/nightlight/nightlight.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python
+
+import time
+from ltr559 import setup, update_sensor, get_lux, get_proximity
+from rgbmatrix5x5 import RGBMatrix5x5
+
+print("""This Pimoroni Breakout Garden example requires an
+LTR-559 Light and Proximity Breakout and a 5x5 RGB Matrix Breakout.
+This example creates a little nightlight that can be toggled on or
+off by tapping the proximity sensor with your finger, or triggered
+automatically when it's dark. Press Ctrl+C a couple times to exit.
```

Comment:
Near as I can tell it's only necessary to hit `Ctrl+C` once. Should probably drop this onto another line for clarity and rephrase to: `Press Ctrl+C to exit.`